app/views/pages/developer-guide/about-platformos/about-platformos.liquid

```diff
@@ -5,35 +5,30 @@ metadata:
   description: An overview of the main features and technologies of platformOS.
 ---
 
-platformOS is a **model-based application development platform**. Aimed at frontend developers and site builders, it provides for a flexible and limitless API-first development approach while automating the whole DevOps infrastructure.
+platformOS is a **managed application development platform**. Aimed at developers, it provides for a flexible and limitless API-first development approach while automating the whole DevOps infrastructure.
```

Comment:
What does managed application development platform means? It might be a term I'm not familiar with, but by reading this I'm not sure what platformOS is :)

Also, I would rather say "designed" for developers instead of aimed.